docs/dsp-tools-xmlupload.md

```diff
@@ -793,6 +789,100 @@ Example:
 ```
 
 
+## DSP base resources / base properties to be used directly in the XML file
+There is a number of base resources and base properties that must not be subclassed in a project ontology. They are 
+directly available in the XML data file. Please have in mind that built-in names of the knora-base ontology must be used 
+without prepended colon.  
+See also [the related part of the ontology documentation](dsp-tools-create-ontologies.md#dsp-base-resources-base-properties-to-be-used-directly-in-the-xml-file)
+
+### `<annotation>`
+`<annotation>` is an annotation to another resource of any class. It must have the following predefined properties:
+
+- `hasComment` (1-n)
+- `isAnnotationOf` (1)
+
+Example:
+```xml
+<annotation label="Annotation to another resource" id="annotation_0" permissions="res-default">
+    <text-prop name="hasComment">
+        <text encoding="utf8" permissions="prop-default">This is an annotation to a resource.</text>
+    </text-prop>
+    <resptr-prop name="isAnnotationOf">
+        <resptr permissions="prop-default">img_1</resptr>
+    </resptr-prop>
+</annotation>
+```
+
+Technical note: An `<annotation>` is in fact a `<resource restype="Annotation">`.
+
+### `<region>`
+A `<region>` resource defines a region of interest (ROI) in an image. It must have the following predefined properties:
+
+- `hasColor` (1)
+- `isRegionOf` (1)
+- `hasGeometry` (1)
+- `hasComment` (1-n)
+
+There are three types of Geometry shapes (rectangle, circle, polygon), but only the rectangle is implemented.
```

Comment:
I'm not sure what value this statement adds. Might be more helpful to just write "the only supported geometry is 'rectangle'."

Reply:
I generally prefer to leave such things in the docs, in order not to forget about them. The API can process circles and polygons, but neither DSP-APP nor Tangoh display them. For D-Scribes (a project that will soon come to us), the polygons will be necessary. So for me, it is important to conserve the knowledge about such things. I don't know if they are documented elsewhere.